dags/examples/xpk_example_dag.py

```diff
@@ -53,3 +54,35 @@
       time_out_in_min=60,
       num_slices=2,
   ).run()
+
+  group_id = "chained_tests"
```

Comment:
Could you add a comment explaining what this demonstrates?

Reply:
Sure thing!